src/Microsoft.Data.SqlClient/tests/ManualTests/AlwaysEncrypted/TestFixtures/SQLSetupStrategyCertStoreProvider.cs

```diff
@@ -11,7 +11,7 @@ public class SQLSetupStrategyCertStoreProvider : SQLSetupStrategy
     {
         public SqlColumnEncryptionCertificateStoreProvider CertStoreProvider;
         public CspColumnMasterKey CspColumnMasterKey;
-
+    
```

Comment:
Nit: Trailing spaces (the blank line is okay).

Reply:
removed.